### DISCUSSION TITLE
use template to create file will show `[object Object]`

### DISCUSSION THREAD

**Discussion by FlyingTurtleX** [SilentVoid13/Templater#945] [Help]
I am trying to create a new file in a specified folder by this template:

```
<% tp.file.create_new(tp.file.find_tfile("PN-U"), tp.date.now("YYYYMMDDHHmm-"), true, app.vault.getAbstractFileByPath("my folder")) %>
```

But in the first line in the created file shows `[object Object]`. Like this:

<img width="554" alt="image" src="https://user-images.githubusercontent.com/43435304/205529782-c276bb72-9ced-4eb2-a7e0-ee7a4ca23f4b.png">

 If I stop trying to append `true, app.vault.getAbstractFileByPath("my folder")` then it goes away. Anyone know how I can append the text properly and avoid the issue? 

Any help would be appreciated!

**Comment by AB1908** [comment]
What's the use case?

**Reply by FlyingTurtleX** [reply]
The contents of PN-U file are as follows:
```
---
tag: 
encounter: 
created-date: <% tp.file.creation_date("YYYY-MM-DD HH:mm:ss") %>
last-modified-date: <% tp.file.last_modified_date("YYYY-MM-DD HH:mm:ss") %>
---

# 



<% tp.file.cursor(1) %>
```

Thanks for your answer!

**Reply by AB1908** [reply]
Haha that's not what I asked. What is it that you want to do? What does this script accomplish?

**Comment by FlyingTurtleX** [comment]
I want to create the new file in another folder with opening automatically.
